test/test_helper.rb

```diff
@@ -7,7 +7,7 @@
 
 VCR.configure do |config|
   config.cassette_library_dir = "test/vcr_cassettes"
-  config.hook_into :webmock
+  config.hook_into :faraday
```

Comment:
what's this?

Reply:
VCR needs to hook into the http client library we are using, which is faraday. I am not sure why it was using webmock, as I don't see anywhere where webmock is used, but I am guessing it was probably used to stub requests and create the cassettes, and then removed. Because I created the cassette from actually hitting tableau, I needed to set this to faraday.